autograder-core/src/test/resources/de/firemage/autograder/core/check_tests/ClosedSetOfValues/code/Test.java

```diff
@@ -37,12 +37,12 @@ public static void main(String[] args) {
 }
 
 class TestFiniteListing {
-    private static final char[] VALID_ARRAY = {'R', 'x', 'X', '|', '\\', '/', '_', '*', ' '}; // Not Ok
-    private static final List<Character> VALID_LIST = List.of('R', 'x', 'X', '|', '\\', '/', '_', '*', ' '); // Not Ok
-    private static final Set<String> FINITE_SET = Set.of("monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"); // Not Ok
-    private static final char[] NOT_ENOUGH_DISTINCT_VALUES_ARRAY = { 'a', 'b', 'b', 'a' }; // Ok (2 distinct values)
-    private static final List<Character> NOT_ENOUGH_DISTINCT_VALUES_LIST = List.of('a', 'b', 'b', 'a'); // Ok (2 distinct values)
-    private static final Set<String> NOT_ENOUGH_DISTINCT_VALUES_SET = Set.of("monday", "monday", "monday", "monday"); // Ok (1 distinct value)
+    private static final char[] VALID_ARRAY = {'R', 'x', 'X', '|', '\\', '/', '_', '*', ' '}; /*@ not ok @*/
+    private static final List<Character> VALID_LIST = List.of('R', 'x', 'X', '|', '\\', '/', '_', '*', ' '); /*@ not ok @*/
+    private static final Set<String> FINITE_SET = Set.of("monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"); /*@ not ok @*/
+    private static final char[] NOT_ENOUGH_DISTINCT_VALUES_ARRAY = { 'a', 'b', 'b', 'a' }; //?# ok (2 distinct values)
```

Comment:
What is the `//?#` syntax? This should be documented in `Test_Framework.md` and why is `/*@` not used here? (I like the `//?#` more, because it is easier to write)

Reply:
//?# is an artifact of an older version of the syntax which I discarded in favor of the /*@ ... @*/ syntax. The latter has the benefit that it has a defined end tag and is a valid Java comment, so it doesn't break the syntax highlighting in your editor (//?# is also valid Java, but cannot be used to mark multiple problems in a single line). Do you have any suggestions on how to improve the syntax (maybe /*# ... #*/)?

Reply:
`/*@ @*/` is more work to type than `//?#`. I use a standard german keyboard, so for `@` I have to press the `Alt + Gr` and `Q` keys, which are on opposite ends of the keyboard.

I like the suggestion to use `/*# #*/` which would be easier to type. If it is not much work, it would be nice to have support for `//#` as well. In most cases one does not have multiple problems in the same line.